mirdata/utils.py

```diff
@@ -208,10 +208,10 @@ def load_json_index(filename):
 
```

Comment:
this file should not be included in this PR

Reply:
Black modifies it.